lightning/src/ln/peer_handler.rs

```diff
@@ -665,7 +666,12 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref> PeerManager<D
 			let mut msgs_to_forward = Vec::new();
 			let mut peer_node_id = None;
 			let pause_read = match peers.peers.get_mut(peer_descriptor) {
-				None => panic!("Descriptor for read_event is not already known to PeerManager"),
```

Comment:
What separates these current panics that we're removing from the panic in `process_events`? Should we eventually aim to get rid of that `panic` as well?

Reply:
Hmm, the only panic in `process_events` is the one for "the `peers` man and the `node_id_to_descriptor` map is out of sync" which I don't think should ever happen with the new code, unless the users' `SocketDescriptor::eq or hash` functions don't work right, which I think we should still panic on.